components/dashboard/src/start/StartPage.tsx

```diff
@@ -139,3 +142,16 @@ function StartError(props: { error: StartWorkspaceError }) {
     }
     return <p className="text-base text-gitpod-red w-96">{error.message}</p>;
 }
+
+function OrgDefaultWorkspaceWarningView(props: { workspaceId: string }) {
+    const { data: imageInfo } = useDefaultWorkspaceImageQuery(props.workspaceId);
+    if (imageInfo?.source !== "organization") {
+        return null;
+    }
+    return (
+        <Alert className="w-96 mt-4" type="warning">
+            A custom <span className="font-medium">default workspace image</span> is set for this organization. Contact
+            an organization owner or specify a different image in <code>.gitpod.yml</code>.
+        </Alert>
```

Comment:
Yep it would be better, but we can't identify if error is coming from image setting or not 😂. 

i.e. It can stop workspace and show error when the disk space is too large or some other reasons

Reply:
I change to

> **Could not use workspace image?** Try a different workspace image in the yaml configuration or check the default workspace image in organization settings.